src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<Configuration status="info">
+    <Appenders>
+        <Console name="LogToConsole" target="SYSTEM_OUT">
+            <PatternLayout pattern = "%d{HH:mm:ss.SSS} %-5lever %logger{36} - %msg%n"/>
```

Comment:
why lever? does it work? please add screenshot from console that you have logs there